src/mock-data.js

```diff
@@ -1,43 +0,0 @@
-export default [
```

Comment:
i'm curious where this comes from? did you start this repo from an existing repo with this file?

Reply:
Nope, actually I searched for swiping cards in React yesterday & borrow this mock-data from this repo [swiping card react](https://github.com/GavBaros/react-tinder-cards/blob/master/src/data.js)